build.sbt

```diff
@@ -13,16 +13,20 @@ inThisBuild(List(
   )
 ))
 
-lazy val `sbt-compatibility-rules` = project
+lazy val root = (project in file("."))
+  .aggregate(`sbt-compatibility-rules`, `sbt-compatibility`)
```

Comment:
You may need to aggregate `sbt-compatibility-dummy` too. It's used to get the version via a Maven Central badge in the README.

Reply:
Addressed in 17cc75674d12b7d3ff4c3943af215adc9bd6702c